compiler/src/dotty/tools/dotc/parsing/Parsers.scala

```diff
@@ -401,7 +401,9 @@ object Parsers {
       false
     }
 
-    def errorTermTree: Literal = atSpan(in.offset) { Literal(Constant(null)) }
+    def errorTermTree: Tree = atSpan(in.offset) {
+      TypedSplice(Literal(Constant(null)).withType(Types.UnspecifiedErrorType))
```

Comment:
I think a possibly better fix would be to use `_root_.scala.Predef.???` instead of `null`.

Reply:
Unfortunately, the following new follow-on error appears in some cases
```
Stable identifier required, but _root_.scala.Predef.??? found
```

Reply:
In general, if the term we generate has type `Nothing` it generates follow-on error that we should avoid.

Reply:
It looks like we will need to fix the language server `signatureHelp`.